src/module-elasticsuite-core/Index/Mapping/Field.php

```diff
@@ -320,21 +320,20 @@ private function getMultiFieldMappingPropertyConfig($analyzers)
      *
      * @return array
      */
-    private function getFieldAnalyzers()
+    private function getFieldAnalyzers(): array
     {
         $analyzers = [];
 
         if ($this->isSearchable() || $this->isUsedForSortBy()) {
             // Default search analyzer.
             $analyzers = [$this->getDefaultSearchAnalyzer()];
-
-            if ($this->isSearchable() && $this->getSearchWeight() > 1) {
-                $analyzers[] = self::ANALYZER_WHITESPACE;
-                $analyzers[] = self::ANALYZER_SHINGLE;
-            }
+        }
```

Comment:
Why closing the brace here ?

Reply:
Condition "$this->isSearchable()" was duplicated in previous logic construction http://joxi.ru/ZrJ0V44iwxM9P2 . When I changed it - the code became clean http://joxi.ru/12MN144ily0lMA